rest-api/offline/bigquery_sync.py

```diff
@@ -92,6 +92,7 @@ def rebuild_bq_participant_task(timestamp, limit=0):
   mod_bqgen = BQPDRQuestionnaireResponseGenerator()
 
   with dao.session() as session:
+    count = 0
     while limit:
       limit -= 1
       # Collect all participants who do not have a PS generated yet or the modified date is less than the timestamp.
```

Comment:
I just wanted to try the new multi line comment feature.

Reply:
Cool.